### DISCUSSION TITLE
Own subdomain as proxy

### DISCUSSION THREAD

**Discussion by urlexnilz** [johannschopplich/nuxt-gtag#39] [General]
Hi,
I just noticed that "nuxt-gtag" is using https://www.googletagmanager.com/gtag/js?id=${id} (hardcoded in code).  We want to use a docker image as proxy for the domain www.googletagmanager.com. Is it possible to change the domain via parameter?

```
// Inject the Google Analytics script.
useHead({
script: [{ src: `https://www.googletagmanager.com/gtag/js?id=${id}` }],
})
```

**Comment by johannschopplich** [comment]
A custom domain is not supported at the moment, but PR happily welcome!

**Comment by johannschopplich** [comment]
Your feature request was implemented in https://github.com/johannschopplich/nuxt-gtag/commit/2c4075cf9d1ad814b777fa36f8ea1620dac9a81a. Will be released in the next version.


Usage: 

| Option | Type | Default | Description |
| --- | --- | --- | --- |
| `url` | `string` | `'https://www.googletagmanager.com/gtag/js'` | The URL to the `gtag.js` script. Use this option to load the script from a custom URL. |
